front-end/src/screens/OnChain/Motions/query.ts

```diff
@@ -0,0 +1,66 @@
+import gql from 'graphql-tag';
+
+// post_type.ON_CHAIN === 2
+// post_topic.COUNCIL === 2
+
+export const QUERY_LATEST_MOTIONSS = gql`
+  query LatestMotionPosts($postType: Int! = 2, $postTopic: Int! = 2, $limit: Int! = 5 ) {
```

Comment:
shouldn't we keep postType 2, postTopic 2 in constant const ON_CHAIN_TYPE = 2 const COUNCIL_TOPIC = 2 so it is more clear what the numbers means

Reply:
The comments above the query tries to help, but I agree it's not perfect. Also I did not want to set this as a variable because it's not supposed to change. There was unfortunately no way for me to use a variable as default.

Reply:
Should be better now. The names are not entirely generic now though unlike the query, but i guess it's fine..